native-stack/README.md

```diff
@@ -87,21 +91,29 @@ Whether the back button title should be visible or not. Defaults to `true`. Only
 
 Function which returns a React Element to display on the right side of the header.
 
+#### `headerLeft`
+
+Function which returns a React Element to display on the left side of the header. For now, on Android, using it will cause the title to also disappear.
```

Comment:
@WoLewicki any recommendation on how to implement a custom back button with a title on android? happy to send. PR but needs some guidance on how to proceed

Reply:
https://github.com/software-mansion/react-native-screens/pull/457/files#diff-d63d0fa38279f7aa0d11c045d518a176L242 removes the title. You can comment that line and see what happens and work on it to be handled properly.